### DISCUSSION TITLE
Error while running T_AddToCart class

### DISCUSSION THREAD

**Discussion by bhakarswayamavarapu** [Xceptance/XLT#395] [General]
Hi team,
Getting below error while running T_Add to cart class
**EVENT: TAddToCart_US - FlowStopped - TAddToCart_US - 'SafetyBreak for AddToCartFlow.run reached! Unable to add the desired number of products to the cart**

**Comment by rschwietzke** [comment]
What is the question? If the suite cannot find a product that can be added to the cart, it will give up after a while.
